lib/rest-adapter.js

```diff
@@ -586,7 +609,7 @@ function RestMethod(restClass, sharedMethod) {
   this.notes = sharedMethod.notes;
   this.documented = sharedMethod.documented;
 
-  var methodRoutes = getRoutes(sharedMethod);
+  var methodRoutes = getRoutes(sharedMethod, options);
```

Comment:
Haha, you're right about me adding it. Apologies, I've been commenting from a mini-holiday, brain half-absent. I am happy to implement it in whatever way you deem best. I will update the PR accordingly. Is force pushing a new, single commit the best way to do so?